books/RayTracingInOneWeekend.html

```diff
@@ -581,11 +600,12 @@
 gradient to the color in addition to the vertical gradient.
 
 I then did a standard graphics trick of scaling that to $0.0 ≤ t ≤ 1.0$.
-When $t = 1.0$ I want blue.
-When $t = 0.0$ I want white.
+(In this context, the parameter $t$ is different from the image coordinate $t$ above.)
```

Comment:
Please consider changing the variable from `t`. I'm unhappy with aliasing this early in the series

Reply:
This is one problem with adopting _s_,_t_ as image coordinates, because _t_ as a parametric variable comes up often. I'm certainly open to changing this particular case, but it's elsewhere, I think.

Reply:
Hmmm. When lerping, this is frequently done with α. I'll at least change this case to use _a_.

Reply:
Heh. And then immediately below, the ray-sphere intersection code uses _t_ as the ray-sphere intersection parameter. `¯\_(ツ)_/¯`. I'll leave that one.